Automattic/jetpack
# Admin Page: new Performance tab.

Fixes #10550

#### Changes proposed in this Pull Request:

This PR moves existing cards over to a new "Performance" section in Jetpack settings, and makes that "Performance" tab the new default tab you see when heading to Jetpack > Settings in your dashboard.

![image](https://user-images.githubusercontent.com/426388/50154173-d4e4f400-02c8-11e9-80dc-1ba5678c4df0.png)


#### Testing instructions:

* Go to Jetpack > Settings
* Make sure all settings work, including the new Performance section.

#### Proposed changelog entry for your changes:

* Jetpack Dashboard: add a new "Performance" section
